src/participant/participant.ts

```diff
@@ -890,8 +901,14 @@ export default class ParticipantController {
   }): Promise<string | undefined> {
     const databases = await this._getDatabases({ stream });
 
-    if (databases === undefined || databases.length === 0) {
-      log.error('No databases found');
+    if (databases === undefined) {
+      stream.markdown(
+        vscode.l10n.t('An error occurred when getting the databases.')
```

Comment:
Can we surface the error to users here? Something like permission issues is more transparent then.

Reply:
good point, not sure how to word it right. could something like `"An error occurred when getting the databases. You may lack the necessary permissions.` work or we can be even more explicit?